### DISCUSSION TITLE
Zenith and Nadir markers.

### DISCUSSION THREAD

**Discussion by Keatah** [Stellarium/stellarium#2588] [Q&A] [answered]
Zenith is marked with "Z", shouldn't Nadir be marked differently with an "N"?

**Comment by gzotti** [comment]
"N" may be mistaken to mean "North", therefore "Z'"  was chosen.

**Comment by alex-w** [comment, marked as answer]
Z and Z' are standard designations for Zenith and Nadir points

**Comment by Keatah** [comment]
Didn't notice the " ' ".. thx.
